README.md

```diff
@@ -2,10 +2,13 @@
 
 <!-- badges: start -->
 
-[![R-CMD-check-bioc-devel](https://github.com/microbiome/mia/workflows/R-CMD-check-bioc-devel/badge.svg)](https://github.com/microbiome/mia/actions)
-<!--[![R-CMD-check-bioc](https://github.com/microbiome/mia/actions/workflows/check-bioc.yml/badge.svg)](https://github.com/microbiome/mia/actions/workflows/check-bioc.yml)-->
+[![Platforms](http://bioconductor.org/shields/availability/release/miaViz.svg)](https://bioconductor.org/packages/release/bioc/html/miaViz.html#archives)
+[![rworkflows](https://github.com/microbiome/mia/workflows/rworkflows/badge.svg)](https://github.com/microbiome/mia/actions)
+[![Bioc-release](http://bioconductor.org/shields/build/release/bioc/mia.svg)](http://bioconductor.org/packages/release/bioc/html/mia.html)
```

Comment:
You could also add Bioc devel

Reply:
Well, it seems that these badges have same same name, so user cannot distinguish Bioc devel / Bioc release.

Do not add devel